__tests__/app.test.js

```diff
@@ -86,6 +86,46 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: an array of comments for the given article_id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comments) => {
+          expect(comments).toHaveProperty("comment_id");
+          expect(comments).toHaveProperty("votes");
+          expect(comments).toHaveProperty("created_at");
+          expect(comments).toHaveProperty("author");
+          expect(comments).toHaveProperty("body");
+          expect(comments).toHaveProperty("article_id");
+        });
+      });
+  });
```

Comment:
we'll need another test for Status 200, valid ID, but has no comments responds with an empty array of comments